src/tanks/BackendTank.js

```diff
@@ -10,6 +10,7 @@ class BackendTank {
 	isBot: boolean;
 	userId: string;
 	tankName: string;
+	tankHealth: number;

```

Comment:
also, while I'm here, I'm pretty sure we aren't storing tankHealth on the backend

Reply:
Thank you